parameter_files/fates_params_default.cdl

```diff
@@ -386,7 +386,10 @@ variables:
 		fates_maintresp_leaf_atkin2017_baserate:long_name = "Leaf maintenance respiration base rate parameter (r0) per Atkin et al 2017" ;
 	double fates_maintresp_leaf_ryan1991_baserate(fates_pft) ;
 		fates_maintresp_leaf_ryan1991_baserate:units = "gC/gN/s" ;
-		fates_maintresp_leaf_ryan1991_baserate:long_name = "Leaf maintenance respiration base rate per Ryan et al 1991" ;
+		fates_maintresp_leaf_ryan1991_baserate:long_name = "Leaf maintenance respiration base rate per Ryan et al 1991" ;	
+        double fates_maintresp_leaf_decay(fates_pft) ;                                                                          
```

Comment:
I like the introduction of the parameter, but when I see the word "decay" I think of a biological decomposition process. I wonder if "attenuation" would work here, or something similar that conveys how the rate changes over depth.   fates_maintresp_vert_atten?   fates_maintresp_vert_scaler ?

Reply:
I've changed it to maintresp_leaf_vert_scaler_coeff1 and maintresp_leaf_vert_scaler_coeff2.